src/WebJobs.Script.WebHost/WebJobsScriptHostService.cs

```diff
@@ -47,6 +49,10 @@ public class WebJobsScriptHostService : IHostedService, IScriptHostManager, ISer
         private readonly SemaphoreSlim _hostStartSemaphore = new SemaphoreSlim(1, 1);
         private readonly TaskCompletionSource<bool> _hostStartedSource = new TaskCompletionSource<bool>();
         private readonly Task _hostStarted;
+        private readonly IOptions<FunctionsHostingConfigOptions> _hostingConfigOptions;
```

Comment:
Where is FunctionsHostingConfig options coming from?

Reply:
It's injected into the constructor via DI -- but ultimately it gets created by parsing a file that Antares writes for us here: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Config/FunctionsHostingConfigSource.cs#L24. 

That file gets turned into a dictionary, which goes into `IConfiguration`... which then is used to create us this `Options` object via DI.

Reply:
This file has key/value pairs of "flags" that we can set on a stamp-by-stamp value. They're sometimes important to how we decide to specialize, etc, so wanted to make sure we could see all available flags in one place (we currently don't log them).